articles/policies/rate-limits.md

```diff
@@ -209,6 +209,23 @@ The following Auth0 Authentication API endpoints return rate limit-related heade
     </tr>
   </thead>
   <tbody>
+  <tr>
+    <td rowspan="3">Authentication and authorization</td>
+    <td rowspan="3">/authorize</td>
+    <td>IP</td>
+    <td>Non-Free (*)</td>
```

Comment:
I also considered "Paid production". Thoughts?

Reply:
I think "Paid production" is clearer.  "Non-Free (*)" is okay, but combining both "Non" and the asterisk confused me for a second.